commit a9c556ccad819869a6a5d932aac0a75a99372f08
Author: Barry Warsaw <barry@python.org>
Date:   Wed Sep 17 19:32:43 2014 +0300

    _init_completion: Handle cword < 0 (LP: #1289597)
    
    Previously only bash 4.3 seemed to provoke this, but now with the
    empty command consistency tweak it occurs with earlier as well.

diff --git a/bash_completion b/bash_completion
index 7e01ae4..3bb4bc2 100644
--- a/bash_completion
+++ b/bash_completion
@@ -727,7 +727,7 @@ _init_completion()
         fi
     done
 
-    [[ $cword -eq 0 ]] && return 1
+    [[ $cword -le 0 ]] && return 1
     prev=${words[cword-1]}
 
     [[ ${split-} ]] && _split_longopt && split=true
